metastore/rdbms-metastore/src/main/java/org/apache/drill/metastore/rdbms/RdbmsMetastore.java

```diff
@@ -117,6 +117,8 @@ private HikariDataSource dataSource(DrillConfig config) {
   private void initTables(DataSource dataSource) {
     try (Connection connection = dataSource.getConnection()) {
       JdbcConnection jdbcConnection = new JdbcConnection(connection);
+      // TODO It is recommended to use the new function if the following issue is resolved.
+      // https://github.com/liquibase/liquibase/issues/2349
```

Comment:
@luocooong Thanks for this.  Would you mind creating a JIRA issue for this and listing it in the comments?

Reply:
Good idea.